RecoParticleFlow/PFClusterTools/test/ProducePFCalibrationObject.cc

```diff
@@ -57,11 +57,22 @@ void ProducePFCalibrationObject::beginRun(const edm::Run& run, const edm::EventS
   functType["PFfb_ENDCAP"] = PerformanceResult::PFfb_ENDCAP;
   functType["PFfc_BARREL"] = PerformanceResult::PFfc_BARREL;
   functType["PFfc_ENDCAP"] = PerformanceResult::PFfc_ENDCAP;
+  //New added functions for H/EH hadrons
+  functType["PFfaEta_BARRELH"] = PerformanceResult::PFfaEta_BARRELH;
+  functType["PFfaEta_ENDCAPH"] = PerformanceResult::PFfaEta_ENDCAPH;
+  functType["PFfbEta_BARRELH"] = PerformanceResult::PFfbEta_BARRELH;
+  functType["PFfbEta_ENDCAPH"] = PerformanceResult::PFfbEta_ENDCAPH;
+  functType["PFfaEta_BARRELEH"] = PerformanceResult::PFfaEta_BARRELEH;
+  functType["PFfaEta_ENDCAPEH"] = PerformanceResult::PFfaEta_ENDCAPEH;
+  functType["PFfbEta_BARRELEH"] = PerformanceResult::PFfbEta_BARRELEH;
+  functType["PFfbEta_ENDCAPEH"] = PerformanceResult::PFfbEta_ENDCAPEH;
+  //Left older functions untouched for backward compatibility 
   functType["PFfaEta_BARREL"] = PerformanceResult::PFfaEta_BARREL;
   functType["PFfaEta_ENDCAP"] = PerformanceResult::PFfaEta_ENDCAP;
   functType["PFfbEta_BARREL"] = PerformanceResult::PFfbEta_BARREL;
   functType["PFfbEta_ENDCAP"] = PerformanceResult::PFfbEta_ENDCAP;
 
+
```

Comment:
hello @spandeyehep 

I see that in the new payload you've put forward for inclusion in global tags,
3007 3009 3008 3010
and these four:

>     functType["PFfaEta_BARREL"] = PerformanceResult::PFfaEta_BARREL;
>    functType["PFfaEta_ENDCAP"] = PerformanceResult::PFfaEta_ENDCAP;
>    functType["PFfbEta_BARREL"] = PerformanceResult::PFfbEta_BARREL;
>    functType["PFfbEta_ENDCAP"] = PerformanceResult::PFfbEta_ENDCAP;

are not used anywhere anymore, other than " for backward compatibility " statements.
What backward compatibility are you trying to safeguard ?